aim/aim_manager.py

```diff
@@ -150,6 +151,21 @@ def find(self, context, resource_class, **kwargs):
                 self._make_resource(context.db_session, resource_class, obj))
         return result
 
+    def get_status(self, context, resource):
```

Comment:
I'm curious why this is a separate AimManager call that has to be made after the Resource is retrieved, and not available as an attribute of Resource?